src/test/java/organice/logic/parser/ListCommandParserTest.java

```diff
@@ -0,0 +1,50 @@
+package organice.logic.parser;
+
+import static organice.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import organice.logic.commands.ListCommand;
+import organice.model.person.Type;
+
+public class ListCommandParserTest {
+
+    private ListCommandParser parser = new ListCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsListCommand() {
+
+        // no leading and trailing whitespaces -- no argument given
+        ListCommand expectedListCommand = new ListCommand(null);
+        assertParseSuccess(parser, "list", expectedListCommand);
+
+        // no leading and trailing whitespaces -- patient
+        expectedListCommand = new ListCommand(new Type("patient"));
```

Comment:
A suggestion is to declare constants for Type("patient"), Type("donor"), Type("doctor") and use them in your tets.

Reply:
Added constants.